tests/test_nn.py

```diff
@@ -26,6 +27,7 @@ def test_direct_shape(J, Q, T, N, padding_mode):
         T=T,
         in_channels=C,
         padding_mode=padding_mode,
+        include_lp=include_lp,
     )
     y = graph(x)
     assert y.shape[:2] == (B, Q*J)
```

Comment:
i don't understand. Shouldn't `y.shape[1]` become `Q*J+1` for `include_lp=True`? since we have an extra path